fachhochschule-vorarlberg-author-date.csl

```diff
@@ -1,33 +1,63 @@
 <?xml version="1.0" encoding="utf-8"?>
 <style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="de-DE">
   <info>
-    <title>Fachhochschule Vorarlberg (German)</title>
-    <title-short>FHV</title-short>
-    <id>http://www.zotero.org/styles/fachhochschule-vorarlberg</id>
-    <link href="http://www.zotero.org/styles/fachhochschule-vorarlberg" rel="self"/>
-    <link href="http://www.fhv.at/bibliothek/schreibzentrum/unterlagen-zum-wissenschaftlichen-arbeiten" rel="documentation"/>
+    <title>Fachhochschule Vorarlberg (author-date, German)</title>
+    <title-short>FHV Autor-Jahr</title-short>
+    <id>http://www.zotero.org/styles/fachhochschule-vorarlberg-author-date</id>
+    <link href="http://www.zotero.org/styles/fachhochschule-vorarlberg-author-date" rel="self"/>
```

Comment:
@rmzelle 
I'm in contact with the user who was struggling with the Github last time and he also provided me with their footnote style now, so we'll need the author-date form.
I've made a pull request for the renamed-styles.json: https://github.com/citation-style-language/styles/pull/2444